core/lib/zksync_core/src/api_server/web3/tests/vm.rs

```diff
@@ -254,6 +262,104 @@ async fn send_raw_transaction_after_snapshot_recovery() {
     .await;
 }
 
+#[derive(Debug)]
+struct SendTransactionOptimisticTest {}
```

Comment:
```suggestion
struct SendTransactionOptimisticTest;
```

Empty structures do not need curlys.